vsintegration/src/FSharp.Editor/CodeFix/ImplementInterfaceCodeFixProvider.fs

```diff
@@ -118,7 +118,7 @@ type internal FSharpImplementInterfaceCodeFixProvider
                             title,
                             (fun (cancellationToken: CancellationToken) ->
                                 async {
-                                    let! sourceText = context.Document.GetTextAsync() |> Async.AwaitTask
+                                    let! sourceText = context.Document.GetTextAsync(cancellationToken) |> Async.AwaitTask
```

Comment:
Why do we shadow `cancellationToken` in other places, but at this line we used the parameter? Maybe a bit of explanation would help me understand :)